### ISSUE TITLE
Quest block break for vegetables doen`s work!

### ISSUE THREAD

**Issue by BierGarten100** [PikaMug/Quests#1808]
**What is your Quests version / build number (do _not_ say "latest")?**
4.0.8-b267

**Describe the problem and how to replicate it. Any console errors to post _using a pastebin_?**
We wanted to add a Quest, where players have to harvest carrots, berries, potatoes and wheat.
We tried it first only with carrots and noticed, that the target was blank (as you can see in the first picture).
Then we tried it with all vegeables that the game offers and noticed that wheat was the only one, that could be seen under the mission target.

However, after breaking carrots, potatoes, berry bushes and wheat the mission shows a completion of everything.
Berries and potatoes even have a different output color than for example wheat (carrots weren't even mentioned in the completion message).

[Image 1 - Quest Accept](https://gyazo.com/453f71f848fce14106e513e1260058e1)

[Image 2 - Quest finished](https://gyazo.com/235042ba2d926e7677d8fecf5051ae7f)

**If applicable, can you provide an example quest from quests.yml _using a pastebin_?**
[Quest - Pastebin](https://pastebin.com/QUgnKK7k)

**Comment by DeadlykillOfficial** [comment]
Make sure you are using correct name for Carrot it's CARROTS and Potato it's POTATOES i believe.

**Comment by BierGarten100** [comment]
We have set the correct names.
See the pictures and the Pastebin Code.

`break-block-names:
          - carrots
          - potatoes
          - sweet_berry_bush
          - wheat`

**Comment by PikaMug** [comment]
What is your server's Minecraft version?

**Comment by BierGarten100** [comment]
> What is your server's Minecraft version?

We have:
CraftBukkit version 3238-Spigot-6c1c1b2-9217b52 (MC: 1.17.1) (Implementing API version 1.17.1-R0.1-SNAPSHOT)

**Comment by PikaMug** [comment]
Thanks. My guess is 0 isn't a valid durability for those blocks. Try setting each to 999 which basically ignores durability.

**Comment by BierGarten100** [comment]
> Thanks. My guess is 0 isn't a valid durability for those blocks. Try setting each to 999 which basically ignores durability.

Alright, but why do you have to change the durabilty for these items, when its working for wheat?

**Comment by BierGarten100** [comment]
> Thanks. My guess is 0 isn't a valid durability for those blocks. Try setting each to 999 which basically ignores durability.

After changing the durability to 999 the mission target shows the amount of vegetables that the player has to harvest.

The problem is, the plugin dosn't count up after breaking one of the vegetables (for example after breaking carrots, the number stayed on 0).

We experimented afterwards and checked the age of the vegetable (in this case carrots).
A full grown carrot has an age of 7. So we used that number in durability and now it worked.
I guess it will work for all the vegetables in the same way. Wheat, carrots and potatoes have the same age after growing fully.
A grown beetroot plant has an age of 3, same for the berry bush.
So to make it work, one has to change the durability to the age of the full grown plant.
